core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -691,6 +691,62 @@ public void minute() {
     assertEquals("minute(\"2020-08-17 01:02:03\")", expression.toString());
   }
 
+  private void testMinuteOfHour(FunctionExpression dateExpression, int minute, String testExpr) {
+    assertEquals(INTEGER, dateExpression.type());
```

Comment:
`assertAll`?

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/196/commits/b3ee7b44118577aa72156856809b533931f81289